s4e-backend/src/main/java/pl/cyfronet/s4e/sync/QueueReceiver.java

```diff
@@ -0,0 +1,21 @@
+package pl.cyfronet.s4e.sync;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.amqp.AmqpRejectAndDontRequeueException;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+
+@RequiredArgsConstructor
+@Slf4j
+public class QueueReceiver {
+    private final SceneAcceptor sceneAcceptor;
+
+    @RabbitListener(queues = "#{incomingQueueName}")
+    public void receive(Notification notification) {
+        log.info(String.format("Handling %s: %s", notification.getEventName(), notification.getObjectKey()));
```

Comment:
I'd argue that we should log that we have received a message from queue. It will by useful to identify problems early on, however, it should probably be done in DEBUG level, you're right.
I'll also correct the message.